<?xml version="1.0" encoding="UTF-8"?>  
<launch>
    <arg name="model_name" default="aruco_visual_marker_26"/>
    <arg name="position_x" default="0"/>
    <arg name="position_y" default="0"/>
    <arg name="position_z" default="0.25"/>

    <arg name="orientation_x" default="0"/>
    <arg name="orientation_y" default="0"/>
    <arg name="orientation_z" default="0"/>
    <arg name="orientation_w" default="1"/>
    
    <node pkg="marker_move" type="teleport" name="teleport" output="screen">
      <param name="Model_name" value="$(arg model_name)"/>
    	<param name="Position_x" value="$(arg position_x)"/>
    	<param name="Position_y" value="$(arg position_y)"/>
    	<param name="Position_z" value="$(arg position_z)"/>

    	<param name="Orientation_x" value="$(arg orientation_x)"/>
    	<param name="Orientation_y" value="$(arg orientation_y)"/>
    	<param name="Orientation_z" value="$(arg orientation_z)"/>
    	<param name="Orientation_w" value="$(arg orientation_w)"/>

    </node>

  </launch> 
